app/routes/questionnaire.py

```diff
@@ -291,14 +290,24 @@ def get_thank_you(schema):
             confirmation_email.handle_post()
             return redirect(
                 url_for(
-                    "post_submission.get_confirmation_email_sent",
+                    ".get_confirmation_email_sent",
                     email=confirmation_email.get_url_safe_serialized_email(),
                 )
             )
 
+    show_feedback_call_to_action = True
+
+    try:
+        Feedback(schema, form_data=request.form)
```

Comment:
This will check the submitted_time twice as its already done in ThankYou

Reply:
I don't think there is a more elegant way to do this without making multiple calls to `get_session_store`

Reply:
Done by moving the check out of the Feedback and ThankYou classes and into the routes before_request method